Makefile

```diff
@@ -20,4 +20,13 @@ clean_workspace:
 .PHONY: check_deps
 check_deps:
 	@# Checks that Node is installed
-	@node -v > /dev/null 2>&1 || (echo -e 'node is not installed\n' && exit 42)
\ No newline at end of file
+	@node -v > /dev/null 2>&1 || (echo -e 'node is not installed\n' && exit 42)
+
+
+build_core: 
+	pushd packages/core && npm run build && popd
+
+build: build_core
```

Comment:
why do you need to write build_core at the end ?

Reply:
That's a way to state that build relies on build_core :)